src/Microsoft.AspNet.Mvc.Razor.Host/ViewHierarchyUtility.cs

```diff
@@ -68,12 +68,6 @@ private static IEnumerable<string> GetHierarchicalPath(string relativePath, stri
                 relativePath = relativePath.Substring(1);
             }
 
-            if (Path.IsPathRooted(relativePath))
```

Comment:
@Eilon, the paths that are used in this API are always app relative (locating `_ViewStart`, `_ViewImports`). This would be a safe change.